sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusListener.cs

```diff
@@ -185,89 +159,89 @@ public void Dispose()
                 // For now, rely on finalization to clean up _cancellationTokenSource's wait handle (if allocated).
                 _cancellationTokenSource.Cancel();
 
-                if (_receiver != null && _receiver.IsValueCreated)
+                if (_batchReceiver != null && _batchReceiver.IsValueCreated)
```

Comment:
We might want to implement IAsyncDisposable.

Reply:
the ServiceBusReceiver already does.